modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -192,6 +192,81 @@ definition {
 			actionLabel = "Action Label");
 	}
 
+	@description = "LPS-178405 - Verify if actions works with oldValue function when trigger is On After Delete (Add an Object Entry)"
+	@priority = 5
+	test CanCreateActionAddAnObjectEntryUsingOldValueWithOnAfterDeleteTrigger {
+		task ("Given: Two custom objects with a text field") {
+			for (var objectNum : list "1,2") {
+				ObjectAdmin.addObjectViaAPI(
```

Comment:
Can you add the ticket number as the custom object name. So like "Custom Object 178405-${objectNum)"